Sources/Flows/OAuth2.swift

```diff
@@ -489,7 +489,57 @@ open class OAuth2: OAuth2Base {
 			callback(nil, error.asOAuth2Error)
 		}
 	}
-	
+    
+    // MARK: - Exchange Access Token For Resource
+    open func tokenRequestForExchangeAccessTokenForResource(resourceId: String, resourceType: String, params: OAuth2StringDict? = nil) throws -> OAuth2AuthRequest {
+        guard let accessToken = clientConfig.accessToken, !accessToken.isEmpty else {
+            throw OAuth2Error.noAccessToken
+        }
+        guard let resourceUrl = clientConfig.resourceURL, !resourceUrl.absoluteString.isEmpty else {
+            throw OAuth2Error.noResourceURL
+        }
+        let req = OAuth2AuthRequest(url: (clientConfig.tokenURL ?? clientConfig.authorizeURL))
```

Comment:
Code-style consistency: missing empty line
```suggestion
        }
 
        let req = OAuth2AuthRequest(url: (clientConfig.tokenURL ?? clientConfig.authorizeURL))
```